doc/Uplink.xml

```diff
@@ -292,20 +292,20 @@
       <para>Note that for the following discussion the roles of client and server are swapped.</para>
       <para>The remote client shall authenticate itself using a valid server certificate. The local
         service shall verify the validity of the remote certificate according to RFC 6125.</para>
+      <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
       <section>
         <title>mTLS authentication</title>
+        <para>Device signaling support for mTLS based authentication shall include <literal>mTLS</literal> in the list of <literal>AuthorizationModes</literal> of GetServiceCapabilities response.</para>
         <para>When using mTLS authentication, the local service shall authenticate itself at the
           remote client using TLS client authentication according to RFC 5246 or subsequent
           specifications.</para>
         <para>To uniquely identify a local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
-        <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
       </section>
       <section>
-        <title>JWT authentication</title>
-        <para>When using JWT authentication, the local service shall provide a JWT token when sending the upgrade request.</para>
-        <para>The remote client shall validate the JWT token provided following the JWT specification as defined in the Core specification.</para>
-        <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers.</para>
-        <para>When a device indicates JsonWebToken is supported through its Capabilities, then it shall support JWT authentication in this specification.</para>
+        <title>Access token authentication</title>
+        <para>Device signaling support for access token based authentication shall include <literal>AccessToken</literal> in the list of AuthorizationModes of GetServiceCapabilities response. Device shall signal value greater than 0 for MaxConfigurations in AuthorizationServer.GetServiceCapabilities response from the Security service.</para>
```

Comment:
> **AuthorizationServer.GetServiceCapabilities**

Shouldn't this be GetServiceCapabilities.AuthorizationServer (outer to inner) OR  AuthorizationServerConfigurationCapabilities (specific element) ?

Am I missing something?

Reply:
No, it is simply wrong. I'm going to update it.

Reply:
New wording is now `0 for AuthorizationServer.MaxConfigurations in GetServiceCapabilities`.